ci/cron/src/Main.hs

```diff
@@ -150,19 +159,20 @@ build_docs_folder path versions latest = do
                 then do
                     putStrLn "  Checks, reusing existing."
                     copy (old </> version) $ new </> version
-                    return $ Just version
+                    return $ Just gh_version
                 else do
                     putStrLn "  Check failed. Rebuilding..."
                     build version new
-                    return $ Just version
+                    return $ Just gh_version
             else do
                 putStrLn "  Not found. Building..."
                 build version new
-                return $ Just version)
+                return $ Just gh_version)
         putStrLn $ "Copying latest (" <> latest <> ") to top-level..."
         copy (new </> latest </> "*") (new <> "/")
         putStrLn "Creating versions.json..."
-        create_versions_json documented_versions new
+        create_versions_json (filter (not . prerelease) documented_versions) (new </> "versions.json")
+        create_versions_json (filter prerelease documented_versions) (new </> "snapshots.json")
```

Comment:
Is `snapshots.json` used for anything atm or is this just preparation so we could use it in the future?

Reply:
The script used to decide whether to run or not based on whether the most recent published release had changed. If we want to publish documentation for snapshot releases that won't work anymore.

Because we do not want prereleases in the drop-down, they cannot be in versions.json, but we still need to know which ones we have published already, so we need to keep track of them _somewhere_.

We also need to know, for each docs version we have built, whether we built it as prerelease or not, because if a version changes (we publish a stable release after this has run), we still need to rerun.

This is why we need this snapshots.json, and why it is fetched by `fetch_s3_versions`.

Reply:
Thanks for the explanation! Makes a lot of sense.